idaes/generic_models/unit_models/column_models/solvent_column.py

```diff
@@ -312,32 +312,32 @@ def _make_performance(self):
         self.diameter_column = Var(domain=Reals,
                                    initialize=0.1,
                                    units=pyunits.m,
-                                   doc='Column diameter')
+                                   doc='Column diameter (m)')
```

Comment:
I generally suggest not including the units in the doc string, as it presumes a specific set of units and effectively duplicates the information stored in units.  Thus, it becomes possible for the doc string to become inconsistent with the actual units if one s changed but not the other,

Reply:
Units have been removed from all doc strings, as suggested.